packages/frontend/editor-ui/src/features/ai/chatHub/composables/availableTools.ts

```diff
@@ -0,0 +1,104 @@
+import { type ChatHubAgentTool } from '@n8n/api-types';
+import { type INode, JINA_AI_TOOL_NODE_TYPE, SEAR_XNG_TOOL_NODE_TYPE } from 'n8n-workflow';
+
+export interface ChatHubToolProvider {
+	name: string;
+	description: string;
+	credentialType: string | undefined;
+	tools: Array<{
+		title?: string;
+		node: INode;
+	}>;
+}
+
+export const AVAILABLE_TOOLS: Record<ChatHubAgentTool, ChatHubToolProvider> = {
+	[JINA_AI_TOOL_NODE_TYPE]: {
```

Comment:
I wonder if we should define these at the node level. This way we can add and import tools more generically.

This could be post-p1 refactor too as well.